OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -118,20 +119,24 @@ public Harvester(Actor self, HarvesterInfo info)
 
 		void INotifyCreated.Created(Actor self)
 		{
+			notify = self.TraitsImplementing<INotifyHarvesterAction>().ToArray();
+
 			if (Info.SearchOnCreation)
 				self.QueueActivity(new FindResources(self));
 		}
 
 		void INotifyBuildComplete.BuildingComplete(Actor self)
 		{
+			notify = self.TraitsImplementing<INotifyHarvesterAction>().ToArray();
```

Comment:
This duplication looks bogus.  Can we remover `BuildingComplete` completely from here?